modules/github-bots/sdk/bot.go

```diff
@@ -18,13 +18,13 @@ import (
 	"github.com/kelseyhightower/envconfig"
 )
 
-// Define a type for keys used in context to prevent key collisions.
-type contextKey string
+// ContextKey define a type for keys used in context to prevent key collisions.
+type ContextKey string
```

Comment:
I think we should keep this private, and add a `AttributeFromContext(ctx)` method instead. This lets us change the implementation without potentially breaking folks that depend on it.

Reply:
sounds good, something like this?